development/gcp/pkg/secretmanager/types.go

```diff
@@ -0,0 +1,11 @@
+package secretmanager
+
+import (
+	secretmanager "google.golang.org/api/secretmanager/v1"
+)
+
+// Service contains services for secret manipulation
+type Service struct {
+	Service        *secretmanager.Service
+	VersionService *secretmanager.ProjectsSecretsVersionsService
+}
```

Comment:
Do you really need to keep this struct in the separate file? I would keep it in the `client.go` and rename it into `service.go`